inkcut/core/workbench.py

```diff
@@ -155,6 +155,11 @@ def run(self):
         # Make sure ^C keeps working
         signal.signal(signal.SIGINT, signal.SIG_DFL)
 
+        locale = QtCore.QLocale.system().name()
+        qtTranslator = QtCore.QTranslator()
+        if qtTranslator.load("translations/"+locale):
+            InkcutWorkbench._instance.application._qapp.installTranslator(qtTranslator)
```

Comment:
please use `self` instead of `InkcutWorkbench._instance`

Reply:
done